oh.ps1

```diff
@@ -1075,7 +1074,7 @@ function write_config_files {
 		copy_config_file $CRED_SETTINGS;
 	}
 	if ( $DEMO_DATA -eq "on" ) {
-		copy_config_file $DEMO_CRED_SETTINGS;
+		(Get-Content "$OH_PATH/$OH_DIR/rsc/$DEMO_CRED_SETTINGS.dist") | Set-Content "$OH_PATH/$OH_DIR/rsc/$CRED_SETTINGS"
 	}
 }
```

Comment:
it could be possible that it is demo_data and not portable?

Reply:
No, not possibile.